src/index.html

```diff
@@ -11,8 +11,61 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__img"
+        src="images/image.png"
+        alt="laptopFoto"
+      />
+
+      <h1 class="card__title">
+        APPLE A1419 iMac 27" Retina
+        <br />
+        5K Monoblock (MNED2UA/A)
+      </h1>
+      <p class="card__title-2">Product code: 195434</p>
+
+      <div class="card__rate">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="card__rate-p">Reviews: 5</p>
+      </div>
+
+      <div class="card__price">
+        <p class="card__price-p">Price:</p>
+        <p class="card__price-a">$2,199</p>
+      </div>
+
+      <div class="card__button">
+        <div
+          data-qa="hover"
+          class="card__button-b"
+        >
+          <p>Buy</p>
+        </div>
```

Comment:
```suggestion
        <a
          data-qa="hover"
          class="card__button-b"
        >
          Buy
        </a>
```
redundant additional wrapper
`a` might look strange here, but usually it would be some link. Or use `button`